translations/base-nl.yaml

```diff
@@ -97,23 +97,23 @@ global:
 
     # The suffix for large numbers, e.g. 1.3k, 400.2M, etc.
     suffix:
-        thousands: k
+        thousands: K
```

Comment:
You could also switch to SI (k,M,G,T) if there are no shortenings in dutch

Reply:
there is a shortening, and as I explained, should be an uppercase K, not lowercase

Reply:
Okay